src/js/read.js

```diff
@@ -33,13 +50,10 @@ if (!readJSConfig.cb){ readJSConfig.cb = function(){
         },
         thresholds:{
             viewport:25, //100 points awarded if the DOM node takes up this percentage of the viewport or higher
-            domNode:30, //100 points awarded if the user scrolls past the percentage point of the DOM node
+            domNode:percentagePoint, //100 points awarded if the user scrolls past the percentage point of the DOM node
             readingPoint:400, // if the number of points exceeds this limit than the person has read the article
             domPolling:100, // the number of points to accumulate before doing any calculations on the DOM
-            timeInView:6 //the minimum number of seconds the DOM node needs to be in view
-            //under 5 seconds: Nigel, Ning, Sam, Natasha, Jamie, Richard
-            //5-10 seconds (1-2 sentences): Rob, Kristen, Edric, Alice
-            //about 30 seconds to 1 minute (paragraph): Patrice, Patrick, Dave, Xiaodong
+            timeInView:(wordCount*(percentagePoint/100)/averageReadSpeed) //calculated minimum time necessary to reach the 100% scroll node above
```

Comment:
This concept is a great addition/improvement.
We can work on how to account for articles with non textual items in it later on.